City-of-Pori/pori-kada
# PORI-355 Fixed caption styling on images on content area

To test:

1. drush cc all
2. Go to edit for example this news item: https://local.pori.fi/uutinen/2018-02-23_paljasta-salaisuutesi-yyterin-salaisuudet-halutaan-julki and add an image on the content area. Make sure it is presented using content half scald context and the alignment is left. Make sure the caption looks nice. Screenshot of what you should see is attached.

![screen shot 2018-03-07 at 16 02 51](https://user-images.githubusercontent.com/2276077/37096374-189ae166-2221-11e8-97fc-7d39117025e7.png)
